dev/NumberBox/NumberBox.cpp

```diff
@@ -300,9 +308,8 @@ void NumberBox::StepValue(bool sign)
         UpdateTextToValue();
         return;
     }
-
     // Input Overwriting - Coerce to min or max
-    if (BasicValidationMode() == winrt::NumberBoxBasicValidationMode::InvalidInputOverwritten && GetBoundState(newVal) != BoundState::InBounds)
+    else if (BasicValidationMode() == winrt::NumberBoxBasicValidationMode::InvalidInputOverwritten && GetBoundState(newVal) != BoundState::InBounds)
```

Comment:
I think it's mostly intentional because MinMaxMode Wrapping should take precedence over coercing the value to the Min/Max. That said the early return is probably not necessary and I'll remove it.